scss/modules/AI-ALLY.scss

```diff
@@ -0,0 +1,170 @@
+header {
+  background-color: rgb(255, 255, 255);
+  width: 100vw;
+  height: 5.5em;
+  border-bottom: 1px solid rgb(236, 240, 241);
+  box-shadow: 0px 4px 8px rgba(0, 0, 0, 0.1);
+  position: fixed;
+  z-index: 5000;
+
+  .row {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 75rem;
+    top: 1rem;
+    position: relative;
+    left: 1rem;
+
+    .center-header {
+      ul {
+        display: flex;
+        gap: 20px;
+        padding: 0px;
+        margin: 0px;
+        li a:hover {
+          background-color: none;
+          color: $GREEN-BUTTON-BASKET;
+        }
+      }
+    }
+
+    .left-haeder {
+      display: flex;
+
+      img {
+        position: relative !important;
```

Comment:
why `!important` we should not use !important as we can. try to find a better way